highlight/highlight
# Create partitioned trace_by_id table

## Summary
Search multiple traces by id is slow since it needs to search many parts in clickhouse. Create a new table to partition the data by the first character of the UUID

## How did you test this change?
1) Load Clickhouse locally (localhost:8123/play)
2) Confirm the `traces_by_id` and `traces_by_id_new` table match

## Are there any deployment considerations?
1) Deploy to create new table and materialized view
2) Find latest time stamp and copy over data from old table
3) Exchange tables

## Does this work require review from our design team?
N/A
